tests/expected_tests.cc

```diff
@@ -0,0 +1,39 @@
+#include "gtest/gtest.h"
+#include "stout/expected.h"
+
+stout::expected<int, std::string> divide(int divident, int divider) {
```

Comment:
Do we need the `, std::string` here? I thought that stout's error type defined in `expected.h` was `std::string` by default.

Reply:
You are right Alex, by default it has `std::string` type, so there is no need to specify it here! Thanks!